ReleaseNotes.md

```diff
@@ -1,3 +1,60 @@
+### New in 0.20.0 (released 2016/06/15)
+
+**Features**
+
+Pagination support has been added, letting you control how much data you want
```

Comment:
Ah, after looking down, I see that "Pagination Support" is being called out in a special manner and isn't in the bulleted list. Maybe we should call out why. For example:

> The big focus of this release is pagination support. Pagination support lets you control...